src/entries/popup/components/CommandK/useSearchableTokens.ts

```diff
@@ -46,12 +50,35 @@ export const useSearchableTokens = (searchQuery: string) => {
   const { hideSmallBalances } = useHideSmallBalancesStore();
   const navigate = useRainbowNavigate();
 
+  const query = searchQuery.toLowerCase();
+
+  const { testnetMode } = useTestnetModeStore();
+
   const { data: searchedAssets, isFetching: isFetchingSearchedAssets } =
     useTokenSearchAllNetworks({
       ...VERIFIED_ASSETS_PAYLOAD,
-      query: searchQuery,
+      query,
     });
 
+  const enableSearchChainAssets = isAddress(query) && !testnetMode;
```

Comment:
why not testnet?

Reply:
Backend team said that they don't support testnet tokens :/